src/drivers/code_hosting/gitlab.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+
+function create_pull_request {
+  local repository=$1
+  local branch=$2
+  local parent_branch=$3
+
+  local to_compare
+  if [ "$parent_branch" = "$MAIN_BRANCH_NAME" ]; then
+    # Allow GitLab to redirect to the proper place if this repository is a fork
+    to_compare="$branch"
```

Comment:
@kevgo it is existing tests. For example (note each failure happened during different rake runs)

```bash
Failing Scenarios:
cucumber -p parallel features/git-town-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature:40 # Scenario: undo
```
```bash
Failing Scenarios:
cucumber -p parallel features/git-town-sync/all_branches/perennial_branch_rebase_tracking_branch_conflict/first_branch.feature:73 # Scenario: continuing without resolving the conflicts
```